pom.xml

```diff
@@ -229,7 +229,7 @@
                 <version>3.0-beta-02</version>
                 <configuration>
                     <preloadSources>
-                        <source>classpath/META-INF/resources/webjars/angularjs/${angularjs-version}/angular.min.js">
+                        <source>classpath/META-INF/resources/webjars/angularjs/${angularjs-version}/angular.min.js"&gt;
```

Comment:
I realize this PR is closed, but it's interesting that this entity substitution for the greater-than symbol didn't break the pom. Hmm.